README.md

```diff
@@ -124,24 +124,15 @@ docker run --publish 9000:9000 --env X_API_KEY=test-secret eldarioninc/pdf-servi
 This project uses `isort` for import sorting, `flake8` for Python linting, and various `eslint plugins` for Javascript linting.
 
 ### Passing Builds via Github
-##### Python Linting
```

Comment:
It might be useful to keep these notes around in case a dev wants to run one of these steps manually prior to commit. Maybe stick them in the wiki on a developer notes page?

Reply:
Great idea! I will add the notes to the wiki!